executor/cleaner/cleaner_test.go

```diff
@@ -95,9 +111,10 @@ func markJobSuccessful(job k8upv1a1.JobObject) {
 	job.SetStatus(k8upv1a1.Status{
 		Conditions: []metav1.Condition{
 			{
-				Type:   k8upv1a1.ConditionCompleted.String(),
-				Status: metav1.ConditionTrue,
-				Reason: k8upv1a1.ReasonSucceeded.String(),
+				Type:               k8upv1a1.ConditionCompleted.String(),
+				Status:             metav1.ConditionTrue,
+				Reason:             k8upv1a1.ReasonSucceeded.String(),
+				LastTransitionTime: metav1.Now(),
 			},
 		},
 	})
```

Comment:
There's also a ts.SetCondition() helper, in case that is more useful

Reply:
`ts.SetCondition()` would need an object already present in the test suite. I prefer the flow of first creating a list of objects and then saving them altogether to the suite.